velox/functions/prestosql/tests/StringFunctionsTest.cpp

```diff
@@ -1361,6 +1361,37 @@ TEST_F(StringFunctionsTest, HmacSha512) {
   EXPECT_EQ(std::nullopt, hmacSha512(std::nullopt, "velox"));
 }
 
+TEST_F(StringFunctionsTest, HmacMd5) {
+  const auto hmacMd5 = [&](std::optional<std::string> arg,
+                              std::optional<std::string> key) {
+    return evaluateOnce<std::string, std::string>(
+        "hmac_md5(c0, c1)", {arg, key}, {VARBINARY(), VARBINARY()});
+  };
+  // Use python hmac lib results as the expected value.
```

Comment:
Why not use Presto results for this ?

Reply:
Will run same in Presto and update the test.